lib/minitest/spec.rb

```diff
@@ -266,6 +266,10 @@ def create name, desc # :nodoc:
         @desc = desc
 
         nuke_test_methods!
+
+        def self.methods_matching re
+          public_instance_methods(false).grep(re).map(&:to_s)
+        end
```

Comment:
This doesn't really make sense here, does it? Maybe I'm confused.

Reply:
The class created by `Class.new` will inherit from Minitest:Spec or some other subclass (Minitest::Spec#spec_type). `Class.new` allows you to run code or create methods for that dynamic subclass which will be defined with the call to describe.

I had initially approached it by changing the method in Runnable, though that would cause issues with inheritance (since we wouldn't want to upset the current expected behavior of a test class inheriting from another running the tests of the inherited class). Thus, we'd need only the classes defined by `describe` to have this behavior, which is why we'd need this here.